host/CMakeLists.txt

```diff
@@ -446,29 +449,13 @@ endif ()
 target_include_directories(oehost PRIVATE ${CMAKE_CURRENT_BINARY_DIR})
 
 if (OE_SGX)
-  # Always link with the EnclaveCommonAPI
-  if (WIN32)
-    set(LIBPATHS ${NUGET_PACKAGE_PATH}/EnclaveCommonAPI/lib/native/x64-Release)
-    set(INCPATHS "${NUGET_PACKAGE_PATH}/EnclaveCommonAPI/Header Files")
-  endif ()
+  # For sgx_enclave_common.h.
+  target_include_directories(oehost PRIVATE
```

Comment:
I would feel much better about this change if we could avoid checking in the PSW header

Reply:
@CodeMonkeyLeet 
We need to have a path for building the SDK for simulation mode, easily, on non-sgx machines. From a usability point of view, installing the Intel DCAP libraries following our documentation [here](https://github.com/openenclave/openenclave/blob/master/docs/GettingStartedDocs/Contributors/WindowsManualSGX1Prereqs.md) or [here](https://github.com/openenclave/openenclave/blob/master/docs/GettingStartedDocs/Contributors/WindowsManualSGX1FLCDCAPPrereqs.md) has been quite painful and error prone if you were to try to setup a Windows development machine to play with the SDK.

I would like to avoid that pain-point that is a barrier to contributing to the SDK.

What benefit do we get from having the user to go through the painful process of getting the enclave common API headers? 

At the very least, we ought to just pull the header from: 
https://github.com/intel/linux-sgx/blob/master/psw/enclave_common/sgx_enclave_common.h

Reply:
Intel plans to publish nuget packages soon right? When they do that can we remove this header and just have people download the nuget packages?

I understand the current process is error prone and painful so I'm with you on that. I would just prefer to move to using the packages when there's a better option available.

Reply:
We already get the sgx_enclave_common.h header from a nuget package and that can be deployed on non-sgx machines, just like the .deb packages for development containing the same header are installed.

Reply:
Why is OE including Intel headers if it doesn't need them for simulation mode builds? Wouldn't this PR be redundant if you just publish an additional Debian package for simulation mode without Intel dependencies?

Reply:
Addressed in PR #3155 by using a wrapper header instead of the TEE header.